### PR TITLE
Please review my pull request

### PR DESCRIPTION
**Pull request by ethyl2**
Please review this, @casey-collab

### CODE REVIEW

File: english/sing/continue/continue-to-sing.md [hunk 1]
```diff
@@ -9,3 +9,9 @@ You must really work out that larynx.
 Do they call you the Iron Lung? Because they should.
 
 You start to feel something in your throat.
+
+You start to think about marshmallows, and s'mores,
+and that prompts you to sing [campfire songs](campfire_songs/rootbeer_song.md).
```
Comment (casey-collab) [inline english/sing/continue/continue-to-sing.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/sing/campfire_songs/rootbeer_song.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+You sing:
```
Comment (casey-collab) [inline english/sing/campfire_songs/rootbeer_song.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ethyl2** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**